modules/gdscript/language_server/godot_lsp.h

```diff
@@ -724,8 +724,10 @@ struct TextDocumentContentChangeEvent {
 
 	void load(const Dictionary &p_params) {
 		text = p_params["text"];
-		rangeLength = p_params["rangeLength"];
-		range.load(p_params["range"]);
+		rangeLength = p_params.get("rangeLength", 0);
+		if (p_params.has("range")) {
```

Comment:
This check will change behavior if `!p_params.has("range")`. Previously, it would have set `range.start` and `range.stop` both to 0. In the new version, it would keep them as they were before. Is this intended?

Edit: I guess according to @HolonProduction's comment, it simply doesn't matter, but I think I would expect `start` and `stop` to be set to 0 if `range` wasn't supplied.